packages/trip-form/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentripplanner/trip-form",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
Hey Binh ... think we might want to bump this back to 0.0.11.  When I execute the lerna stuff, it will increment this to 0.0.12 for the release (that said, I'll probably for a version to that number for all components, so it might not matter).

Reply:
I reverted back to 0.0.11.